src/linear_algebra/clifford_algebra/grading.lean

```diff
@@ -85,14 +85,14 @@ end
 /-- The clifford algebra is graded by the even and odd parts. -/
 instance graded_algebra : graded_algebra (even_odd Q) :=
 graded_algebra.of_alg_hom (even_odd Q)
-  (lift _ $ ⟨graded_algebra.ι Q, graded_algebra.ι_sq_scalar Q⟩)
+  (lift _ ⟨by apply graded_algebra.ι Q, by apply graded_algebra.ι_sq_scalar Q⟩)
```

Comment:
Does filling in the underscore instead also help?
```suggestion
  (lift Q ⟨graded_algebra.ι Q, graded_algebra.ι_sq_scalar Q⟩)
```
Or maybe
```suggestion
  (lift Q ⟨(graded_algebra.ι Q : _), (graded_algebra.ι_sq_scalar Q : _)⟩)
```

Reply:
Replacing `lift _` by `lift Q` doesn't have noticeable effect.

Using `(graded_algebra.ι Q : _)` instead of `by apply graded_algebra.ι Q` is about 0.4s slower.

My tests also showed that omitting the second `by apply` has little effect (maybe +30ms), but omitting the first `by apply` results in ~1s slowdown.

Reply:
Thanks for testing!

Reply:
```suggestion
  -- while not necessary, the by apply makes this elaborate faster
  (lift _ ⟨by apply graded_algebra.ι Q, by apply graded_algebra.ι_sq_scalar Q⟩)
```

Reply:
Interestingly, without the second `by apply`, `(graded_algebra.ι Q : _)` is becomes much slower (+6s) than `by apply graded_algebra.ι Q`.